packages/workflow-core/src/lib/plugins/external-plugin/webhook-plugin.ts

```
@@ -76,4 +76,25 @@
 
     return {};
   }
+
+  async composeRequestHeaders(headers: HeadersInit, context: TContext) {
+    const secrets = await this.secretsManager?.getAll();
+
+    if (secrets && secrets['webhookSharedSecret']) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)